ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,340 @@
+        static int FindMax(double a, double b)
+        {
+            return (int)(a > b ? a : b);
+
+        }
+        static int FindMax(int a, int b)
+        {
+            return a > b ? a : b;
+
+        }
+        static int FindMax(long a, long b)
+        {
+            return (int)(a > b ? a : b);
+
+        }
+        //FindMax 3
+        static int FindMax(double a, double b, double c)
+        {
+            return (int)((a > b) ? (a > c ? a : c) : (b > c ? b : c));
+
+        }
+        static int FindMax(int a, int b, int c)
+        {
+            return (a > b) ? (a > c ? a : c) : (b > c ? b : c);
+
+        }
+        static int FindMax(long a, long b, long c)
+        {
+            return (int)((a > b) ? (a > c ? a : c) : (b > c ? b : c));
+
+        }
+        //FindMax 4
+        static int FindMax(double a, double b, double c, double d)
+        {
+            return (int)((a > b) ? (a > c ? (a > d ? a : d) : (c > d ? c : d)) : (b > c ? (b > d ? b : d) : (c > d ? c : d)));
+        }
+        static int FindMax(int a, int b, int c, int d)
+        {
+            return (a > b) ? (a > c ? (a > d ? a : d) : (c > d ? c : d)) : (b > c ? (b > d ? b : d) : (c > d ? c : d));
+        }
+        static int FindMax(long a, long b, long c, long d)
+        {
+            return (int)((a > b) ? (a > c ? (a > d ? a : d) : (c > d ? c : d)) : (b > c ? (b > d ? b : d) : (c > d ? c : d)));
+        }
+        //FindMin 2
+        static int FindMin(double a, double b)
+        {
+            return (int)(a < b ? a : b);
+
+        }
+        static int FindMin(int a, int b)
+        {
+            return a < b ? a : b;
+
+        }
+        static int FindMin(long a, long b)
+        {
+            return (int)(a < b ? a : b);
+
+        }
+        //FindMin 3
+        static int FindMin(double a, double b, double c)
+        {
+            return (int)((a < b) ? (a < c ? a : c) : (b < c ? b : c));
+        }
+        static int FindMin(int a, int b, int c)
+        {
+            return (a < b) ? (a < c ? a : c) : (b < c ? b : c);
+        }
+        static int FindMin(long a, long b, long c)
+        {
+            return (int)((a < b) ? (a < c ? a : c) : (b < c ? b : c));
+        }
+        //FindMin 4
+        static int FindMin(double a, double b, double c, double d)
+        {
+            return (int)((a < b) ? (a < c ? (a < d ? a : d) : (c < d ? c : d)) : (b < c ? (b < d? b : d) : (c < d? c : d)));
+        }
+        static int FindMin(int a, int b, int c, int d)
+        {
+            return (a < b) ? (a < c ? (a < d ? a : d) : (c < d ? c : d)) : (b < c ? (b < d ? b : d) : (c < d ? c : d));
+        }
+        static int FindMin(long a, long b, long c, long d)
+        {
+            return (int)((a < b) ? (a < c ? (a < d ? a : d) : (c < d ? c : d)) : (b < c ? (b < d ? b : d) : (c < d ? c : d)));
+        }
+
+        static int TrySumIfOdd(int x, int y, out int sum)
+        {
+            if (x % 2 != 0 && y % 2 != 0)
```

Comment:
the task is `returns true if the sum of numbers between inputs is odd, otherwise false`
so first of all you need to sum x and y, and then make a decision about sum is odd.